agent-snapper/src/SnapperAgent.cc

```diff
@@ -520,21 +518,22 @@ YCPValue SnapperAgent::Execute(const YCPPath &path, const YCPValue& arg,
 
 	if (PC(0) == "create") {
 
-            string description  = getValue (argmap, YCPString ("description"), "");
-            string cleanup      = getValue (argmap, YCPString ("cleanup"), "");
-            string type         = getValue (argmap, YCPString ("type"), "single");
-            YCPMap userdata     = getMapValue (argmap, YCPString ("userdata"));
+	    string type = getValue (argmap, YCPString ("type"), "single");
+
+	    SCD scd;
+	    scd.uid = getuid();
+	    scd.description = getValue(argmap, YCPString("description"), "");
+	    scd.cleanup = getValue(argmap, YCPString("cleanup"), "");
+	    scd.userdata = ycpmap2stringmap(getMapValue(argmap, YCPString("userdata")));
```

Comment:
I am not sure if it is problem of github or mixture of tabs and space, but it looks like inconsistent indentation.